db/migrate/20170315095936_update_persistent_volumes_parent_type.rb

```diff
@@ -0,0 +1,8 @@
+class UpdatePersistentVolumesParentType < ActiveRecord::Migration[5.0]
+  class ContainerVolume < ActiveRecord::Base; end
```

Comment:
column ContainerVolume#type exists,
so we have to use
` self.inheritance_column = :_type_disabled # disable STI `